### DISCUSSION TITLE
Exclude a plot from scale (ex: a moving average)

### DISCUSSION THREAD

**Discussion by milanster** [highfestiva/finplot#420] [General]
Plots always seem to be included in scale.
Let's say I'm plotting some candlesticks, volume, and a few moving averages.
If a moving average, say MA200, is far away, the chart will always squeeze everything else to make sure it's displayed. Causing the candles to appear very small and squished.

Is there away to exclude such moving average from scale so that it only appears when it makes sense? Without squishing candlesticks.

Thanks in advance, and for this wonderful library!

**Comment by highfestiva** [comment]
Yes, try `fplt.plot(..., zoomscale=False)`. GL!

**Comment by milanster** [comment]
What's funny is that I tried that out yesterday and it didn't work. And now, just this morning, it works like a charm. Must've been the late-night coding lol
Tack så mycket! And keep up the wonderful work!
